config.php

```diff
@@ -3,13 +3,26 @@
 	# FROTZ_EXE_PATH is the path to your dfrotz executable
 	# required: read permissions
 	#
-	$FROTZ_EXE_PATH 	= '/home/tlef/frotz/dfrotz';
+	$FROTZ_EXE_PATH 	= '/usr/local/bin/dfrotz';
```

Comment:
Your personal path changes also do not be included in the PR. I understand that they need to be configured on your computer, but don't feel like you need to add those to your commit.

Reply:
maybe `$_ENV['HOME'] + '/frotz/dfrotz'` would be better than your personal `tlef` home folder?